code/game/objects/structures/tables_racks.dm

```diff
@@ -562,7 +562,7 @@
 /obj/structure/table/woodentable/attackby(obj/item/I as obj, mob/user as mob, params)
 
 	if (istype(I, /obj/item/stack/tile/grass))
-		del(I)
+		qdel(I)

```

Comment:
Yeah...not going to bother fixing this instance though as it's a non destroy/qdel issue...just wanted to get del calls gone.